packages/web-app-media-viewer/src/App.vue

```diff
@@ -1,100 +1,106 @@
-            controls
-            preload
-          >
-            <source :src="medium.url" :type="`video/${medium.ext}`" />
-          </video>
-          <img
-            v-else
-            :key="`media-image-${medium.id}`"
-            :src="medium.url"
-            :alt="medium.name"
-            :data-id="medium.id"
-            class="uk-box-shadow-medium media-viewer-player"
-          />
-        </div>
-      </transition>
-    </div>
+        <video v-if="medium.isVideo" :key="`media-video-${medium.id}`" controls preload>
+          <source :src="medium.url" :type="`video/${medium.ext}`" />
+        </video>
+        <img
+          v-else
+          :key="`media-image-${medium.id}`"
+          :src="medium.url"
+          :alt="medium.name"
+          :data-id="medium.id"
+        />
+      </div>
+    </transition>
     <div v-if="loading" class="uk-position-center">
-      <oc-spinner :aria-label="$gettext('Loading media')" size="xlarge" />
+      <oc-spinner :aria-label="$gettext('Loading media file')" size="xlarge" />
     </div>
     <oc-icon
       v-if="failed"
       name="review"
       variation="danger"
       size="xlarge"
       class="uk-position-center uk-z-index"
-      :accessible-label="$gettext('Failed to load media')"
+      :accessible-label="$gettext('Failed to load media file')"
     />
 
     <div class="uk-position-medium uk-position-bottom-center media-viewer-details">
+      <p class="oc-text-lead uk-text-center oc-text-truncate oc-p-s media-viewer-file-name">
+        <span class="oc-invisible-sr" v-text="currentMediaLabel" />
+        {{ medium.name }}
+      </p>
       <div
         class="
-          uk-overlay uk-overlay-default
-          oc-p-s
-          uk-text-center
-          oc-text-muted
-          uk-text-truncate
-          media-viewer-file-name
+          oc-background-brand oc-p-s
+          uk-width-large uk-flex uk-flex-middle uk-flex-center uk-flex-around
+          media-viewer-controls-action-bar
         "
       >
-        {{ medium.name }}
-      </div>
-      <div class="uk-overlay uk-overlay-primary uk-light oc-p-s media-viewer-controls-container">
-        <div
-          class="
-            uk-width-large uk-flex uk-flex-middle uk-flex-center uk-flex-around
-            media-viewer-controls-action-bar
-          "
-          style="user-select: none"
+        <oc-button
+          class="media-viewer-controls-previous"
+          appearance="raw"
+          variation="inverse"
+          :aria-label="$gettext('Show previous media file in folder')"
+          @click="prev"
+        >
+          <oc-icon size="large" name="chevron_left" />
+        </oc-button>
+        <p
+          v-if="!$_loader_folderLoading"
+          v-translate="{ displayIndex: activeIndex + 1, availableMediaFiles: mediaFiles.length }"
+          class="oc-m-rm oc-light"
+        >
+          <span class="oc-invisible-sr">You are currently viewing</span> %{ displayIndex } of %{
```

Comment:
It will be more verbose, but please make two separate strings from this. A screen reader and an aria-hidden version. If you extract this current version of the string for transifex, this resolves to:
```
"<span class=\"oc-invisible-sr\">You are currently viewing</span> %{ displayIndex } of %{\n"
"          availableMediaFiles } <span class=\"oc-invisible-sr\">media files in this folder</span>"
```
which is kind of a mess. :-(